romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -87,11 +89,11 @@ def create_image_model(input_model, image_info):
     meta['cal_step']['ramp_fit'] = 'INCOMPLETE'
     meta['photometry'] = testutil.mk_photometry()
     inst = {'meta': meta,
-            'data': data,
+            'data': u.Quantity(data, ru.electron / u.s, dtype=data.dtype),
             'dq': dq,
-            'var_poisson': var_poisson,
-            'var_rnoise': var_rnoise,
-            'err': err,
+            'var_poisson': u.Quantity(var_poisson, ru.electron**2 / u.s**2, dtype=var_poisson.dtype),
+            'var_rnoise': u.Quantity(var_rnoise, ru.electron**2 / u.s**2, dtype=var_rnoise.dtype),
+            'err': u.Quantity(err, ru.electron / u.s, dtype=err.dtype),
```

Comment:
Just thinking about alternatives here... the other way to do this would be to require that image_info have units, and have the equivalent of this code in ramp_fit_step.  I prefer that slightly aesthetically, since ramp_fit_step is closer to knowing what units it's outputting.  It'd _really_ be better for stcal.ramp_fitting to be aware of units, but obviously we're avoiding that route for now.

Reply:
Well, image_info receives the output of stcal, so requiring it to have units could be a pain. Maybe we should ponder this one more in #644 - since those lines kind of relate to unit translation?